src/cljs/rems/administration/create_form.cljs

```diff
@@ -235,11 +240,17 @@
                                          {:value "multiselect", :label (text :t.create-form/type-multiselect)}
                                          {:value "date", :label (text :t.create-form/type-date)}
                                          {:value "attachment", :label (text :t.create-form/type-attachment)}
-                                         {:value "label", :label (text :t.create-form/type-label)}]}])
+                                         {:value "label" :label (text :t.create-form/type-label)
+                                          :on-change #(rf/dispatch [::set-form-field [:items item-index :optional] true])}]}])
 
 (defn- form-item-optional-checkbox [item-index]
-  [checkbox context {:keys [:items item-index :optional]
-                     :label (text :t.create-form/optional)}])
+  (let [item @(rf/subscribe [::form-item item-index])]
+    (if (= "label" (:type item))
+      [checkbox context {:label (text :t.create-form/optional)
+                         :disabled? true}]
```

Comment:
I thought about showing something else here, but a disabled checkbox was really fast to implement, doesn't make the UI jump, and is pretty clear for the user (especially for adminds), so I decided to keep it.